docs/guides/vault/edit-vault-data.md

```diff
@@ -48,7 +50,7 @@ The Vault's owner could be an **EOA**, or any **other smart contract**. In our c
 
 ```typescript title="Setting the URD address in the storage"
 import Web3 from 'web3';
-import constants from "@lukso/lsp-smart-contracts/constants.js";
+import constants from '@lukso/lsp-smart-contracts/constants.js';
```

Comment:
same comment about import for `ERC725YDataKeys` in the guide **accept reject assets**